cmd/subnetcmd/upgradecmd/vm.go

```diff
@@ -0,0 +1,305 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package upgradecmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/ava-labs/avalanche-cli/pkg/binutils"
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanche-cli/pkg/plugins"
+	"github.com/ava-labs/avalanche-cli/pkg/subnet"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/spf13/cobra"
+)
+
+const (
+	futureDeployment  = "All future deployments"
+	localDeployment   = "Existing local deployment (coming soon)"
+	fujiDeployment    = "Fuji"
+	mainnetDeployment = "Mainnet (coming soon)"
+)
+
+var (
+	pluginDir string
+
+	useFuji       bool
+	useMainnet    bool
+	useLocal      bool
+	useFuture     bool
+	useManual     bool
+	useLatest     bool
+	targetVersion string
+	useBinary     string
+)
+
+// avalanche subnet delete
+func newVMCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "vm [subnetName]",
```

Comment:
I think you are expecting to upgrade something else, if not, we don't need subcommands

Reply:
Yes, we also need to add a command to change the upgradeBytes